supervisor/api/host.py

```diff
@@ -222,10 +222,14 @@ async def advanced_logs_handler(
                 "supported for now."
             )
 
-        if request.headers[ACCEPT] == CONTENT_TYPE_X_LOG:
+        if "verbose" in request.query or request.headers[ACCEPT] == CONTENT_TYPE_X_LOG:
```

Comment:
Hm, is `verbose` the right word here? :thinking: 

These are the same logs, just with more metadata no? Maybe we should have named the LogFormatter already different? :thinking: 

From what I can tell we add timestamps currently, maybe just `timestamp` or `timestamped`? :thinking:

Reply:
I named the fields same way as we call them in the CLI. Also, [it's not just timestamp](https://github.com/home-assistant/supervisor/blob/6c1203e4bfcdd49e87398088577fc99850fa01b0/supervisor/utils/systemd_journal.py#L37C12-L59), so that would be a wrong name for the format - it's also host name, journal identifier (which - currently - only varies in host logs) and PID of the process.

Reply:
Hm, ok in that case let's leave as is :+1: